x-pack/test/case_api_integration/common/lib/authentication/roles.ts

```diff
@@ -37,6 +37,8 @@ export const globalRead: Role = {
         feature: {
           securitySolutionFixture: ['read'],
           observabilityFixture: ['read'],
+          actions: ['read'],
```

Comment:
The push case tests need to interact with the actions APIs so they users we created for testing RBAC needs access to actions.